third_party/move/move-vm/runtime/src/storage/ty_layout_converter.rs

```diff
@@ -24,12 +25,7 @@ use move_vm_types::{
         struct_name_indexing::StructNameIndex,
     },
 };
-use once_cell::sync::Lazy;
-use std::sync::Arc;
-
-static OPTION_MODULE_ID: Lazy<ModuleId> =
```

Comment:
Move to language_storage and then also use elsewhere.

Reply:
done